### PR TITLE
Ryver #HELP-1298: Prohibited donor checkbox for ACT

### PR DESCRIPTION
**Pull request by andrew-cormick-dockery**
ACT wish to have a prohibited donor checkbox on their forms just like NSW.  It's a hodgepodge of code though, so I'd appreciate some feedback on approach here.

What I've done is add two new classes `<state>-prohibited-donor` (where `<state>` is currently NSW and ACT), which signals to the JS which text to add to the tooltip next to the checkbox.  Another form-related variable `$add<state>ProhibitedDonor` is created to signal to the template engine what to do.  I guess as other states add their own laws this could be extended in the same way.

I'm still waiting on ACT for confirmation of the exact text to put in that box.

I've tested the AG donation form, NSW donation form and ACT donation form on my dev box and all seems well, but further testing is required to ensure that events and memberships are covered, as well as success/failure.

### CODE REVIEW

File: foreigndonors.php
Comment (johntwyman) [inline foreigndonors.php]: (minor) Upper case "l" in "legislation" to match preceeding line.

Comment (andrew-cormick-dockery) [reply foreigndonors.php]: Or maybe I should just have "ACT electoral legislation", i.e. avoid unnecessary capitals.  I just copied and pasted what NSW had, so I could do it there too.

**Review by johntwyman — APPROVED** [review]
It all looks good to me @andrew-cormick-dockery. You're not wrong there's some squirrely code in here but I can't see anything out of place.

File: foreigndonors.php
Comment (johntwyman) [reply foreigndonors.php]: Yeah it's an incredibly minor issue so I think you're right. The key thing is consistency, rather than "what's technically correct".

**Review by johntwyman — APPROVED** [review]

**Comment by andrew-cormick-dockery** [comment]
OK I've now done the following:

1. Fixed the inconsistencies with capitalisation per John's comment.
2. Ensured that the ACT membership page also has the check
3. ACT paid events also has the check
4. When making a payment, the field "Are you a prohibited donor" is set to No. (I don't think it's possible to set this to yes frankly).

This is good to go.  I don't think we have to wait for Adam's approval on the exact text.

**Pull request merged**
